core/src/banking_stage.rs

```diff
@@ -620,65 +620,50 @@ impl BankingStage {
             )
         });
 
+        // Macro to spawn the scheduler. Different type on `scheduler` and thus
+        // scheduler_controller mean we cannot have an easy if for `scheduler`
+        // assignment without introducing `dyn`.
+        macro_rules! spawn_scheduler {
+            ($scheduler:expr) => {
+                bank_thread_hdls.push(
+                    Builder::new()
+                        .name("solBnkTxSched".to_string())
+                        .spawn(move || {
+                            let scheduler_controller = SchedulerController::new(
+                                decision_maker,
```

Comment:
removed a useless clone here - otherwise this was copied exactly from the previous branches